### DISCUSSION TITLE
What is the distance in profiles? How is it judged?

### DISCUSSION THREAD

**Discussion by Luoboawa** [froobynooby/FarmControl#52] [Q&A] [answered]
What is the distance in profiles? How is it judged?

**Comment by froobynooby** [comment, marked as answer]
The distance metric used is the [Chebyshev distance](https://en.wikipedia.org/wiki/Chebyshev_distance). If two entities are within the configured distance of each other they are considered to be in the same group. Grouping is transitive, so if entity A is within the configured distance of entity B, and entity B is within the configured distance of entity C, then A, B and C are all considered to be in the same group.  For example, a line of several entities could all be considered part of the same group even though the entities at the extreme ends of the line may not be within the configured distance of each other

Optionally, you can ignore the vertical component of the distance calculation. If you'd prefer to group entities by chunk you can also use `same-chunk` in place of the distance to get this effect.

**Comment by Luoboawa** [comment]
Ok, thank you, can I sponsor your project with Alipay?

**Reply by froobynooby** [reply]
Hey, I appreciate the thought but I don't accept any sponsors / donations - thank you though
